index.html

```diff
@@ -0,0 +1,131 @@
+        <li><a id="sponsor-btn" href="#sponsor">Sponsor</a></li>
+        <li><a id="news-btn" href="#">News</a></li>
+        <li class="square"><a id="camp-btn" href="#">CC Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section id="intro" class="intro-section">
+      <h1>SUNSHINE ROCK<br>MUSIC FEST<br>2022</h1>
+      <h2>"Hello! Sunny world"</h2>
+      <div class="text-container">
+        <p>A joyful celebration believing in the value of openes and sharing, creating a positive change from people
+          over 80 countries is taking place in September in Argentina.</p>
+      </div>
+      <h3>2022.09.21(WED) ~ 23(FRI)</h3>
+      <h4>@San Isidro's Racecourse, Buenos Aires, Arg</h4>
+    </section>
+
+    <section id="spaces" class="spaces-section">
+      <h2>Spaces</h2>
+      <div class="horizontal-bar"></div>
+      <section class="desktop-container">
+        <article class="card-container">
+          <i class="bi bi-shop-window"></i>
+          <h3>Workshops</h3>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque, deserunt.</p>
+        </article>
+        <article class="card-container">
+          <i class="bi bi-shop"></i>
+          <h3>Food court</h3>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque, deserunt.</p>
+        </article>
+        <article class="card-container">
+          <i class="bi bi-bag"></i>
+          <h3>Merch</h3>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque, deserunt.</p>
+        </article>
+        <article class="card-container">
+          <i class="bi bi-balloon"></i>
+          <h3>Kids park</h3>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque, deserunt.</p>
+        </article>
+        <article class="card-container">
+          <i class="bi bi-palette"></i>
+          <h3>Art exibition</h3>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque, deserunt.</p>
+        </article>
+      </section>
+      <a class="spaces-mobile-btn" href="#">Join Sunshine Rock MF 2022</a>
+      <a class="spaces-desktop-btn" href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <section id="bands" class="presented-section">
+      <h2>Bands Presented</h2>
+      <div class="horizontal-bar"></div>
+      <section id="cards-container" class="cards-container">
+      </section>
+    </section>
+
+    <section id="sponsor" class="sponsors-section">
+      <h2>Partner</h2>
+      <div class="horizontal-bar"></div>
+      <article class="sponsors-container">
+        <img src="./images/apple-music.svg" alt="Apple Music">
+        <img src="./images/heineken.svg" alt="Heineken">
+        <img src="./images/marshall-logo.svg" alt="Marshall">
+        <img src="./images/redbull-1.svg" alt="RedBull">
+        <img src="./images/spotify-1.svg" alt="Spotify">
+      </article>
+    </section>
+  </main>
+  <footer>
```

Comment:
- [ ] The footer contains a logo and text aligned to the right, however, consider moving the logo more to the left and adding some padding/margin so it is closer to the provided design

![Screenshot_20220519_201236](https://user-images.githubusercontent.com/25227022/169384569-310ee248-ea70-4aec-9770-2db7ba3adc52.png)